DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml

```diff
@@ -2,59 +2,73 @@
+                        <div class="govuk-hint"> Select all that apply. </div>
+                    </fieldset>
+                    <div class="filter-list govuk-checkboxes">
+                        @if (Model.Section.Options != null && Model.Section.Options.Any())
                         {
-                            if (Model.Section.SingleSelectOnly == true)
+                            for (int jj = 0; jj < Model.Section.Options.Count; jj++)
                             {
-                                <div class="filter-option">
-                                    @Html.RadioButtonFor(m => Model.Section.SingleSelectedValue, Model.Section.Options[jj].OptionKey, new { id = $"Section_{jj.ToString()}" })
-                                    <label for="Section_@jj.ToString()">@Model.Section.Options[jj].Name<span>@Model.Section.Options[jj].Description</span></label>
-                                </div>
-                            }
-                            else
-                            {
-                                <div class="filter-option">
-                                    @Html.CheckBoxFor(m => Model.Section.Options[jj].IsSelected, new { @class = (Model.Section.Options[jj].ClearOtherOptionsIfSelected ? "filter-na" : "") })
-                                    <label for="Section_Options_@jj.ToString()__IsSelected">@Model.Section.Options[jj].Name<span>@Model.Section.Options[jj].Description</span></label>
-                                </div>
-                            }
+                                if (Model.Section.SingleSelectOnly == true)
+                                {
+                                    <div class="filter-option govuk-radios__item">
+                                        @Html.RadioButtonFor(m => Model.Section.SingleSelectedValue, Model.Section.Options[jj].OptionKey, new { id = $"Section_{jj.ToString()}", @class = "govuk-radios__input" })
+                                        <label class="govuk-label govuk-radios__label" for="Section_@jj.ToString()">@Model.Section.Options[jj].Name<span>@Model.Section.Options[jj].Description</span></label>
+                                    </div>
+                                }
+                                else if (Model.Section.SectionDataType == "TrainingRoute")
+                                {
+                                    <div class="filter-option govuk-radios__item">
+                                        @Html.RadioButtonFor(m => Model.Section.SingleSelectedValue, Model.Section.Options[jj].OptionKey, new { id = $"Section_{jj.ToString()}", @class = "govuk-radios__input" })
+                                        <label class="govuk-label govuk-radios__label" for="Section_@jj.ToString()">@Model.Section.Options[jj].Name<span>@Model.Section.Options[jj].Description</span></label>
+                                    </div>
+                                }
```

Comment:
What is this special case used for? looks very similar to SingleSelect as it should be

Reply:
This was created for Trainingroute which is used for Covid-19 options which are Radio buttons. There was another SingleSelect  section but I kept them separate so that if we need to style the TrainingRoute on its own it will be easier. I changed that btw